.tarball-release.yml

```diff
@@ -0,0 +1,30 @@
+# The jobs to do a manual DOMjudge release
+createtar:
+  stage: test
+  when: manual
+  image: domjudge/gitlabci:2.1
+  script:
+    - echo -e "\e[0Ksection_start:`date +%s`:buildrelease\r\e[0KBuild of release tarball"
+    - wget https://raw.githubusercontent.com/DOMjudge/domjudge-scripts/master/make_release.sh
+    #- sh ./make_release.sh $DOMJUDGE_VERSION
+    - sh ./make_release.sh 7.3
+    - echo -e "\e[0Ksection_end:`date +%s`:buildrelease\r\e[0K"
+    - echo -e "\e[0Ksection_start:`date +%s`:aptpackages\r\e[0KInstall the expected packages for new release"
+    - apt update
+    - apt install -y acl zip unzip mariadb-server apache2 php php-fpm php-gd php-cli php-intl php-mbstring php-mysql php-curl php-json php-xml php-zip composer ntp
+    - apt install -y make pkg-config sudo debootstrap libcgroup-dev php-cli php-curl php-json php-xml php-zip lsof procps
+    - echo -e "\e[0Ksection_end:`date +%s`:aptpackages\r\e[0K"
+    - echo -e "\e[0Ksection_start:`date +%s`:testtar\r\e[0KTest the release tarball"
+    - tar zxvf domjudge-${DOMJUDGE_VERSION}.tar.gz
+    - cd domjudge-${DOMJUDGE_VERSION}
+    - ./configure
+    - make build
+    - make clean
+    - make all
+    - make clean
```

Comment:
Why do many cleans?

Reply:
To test the different scenarios of the Makefile.

You want me to remove it?

Reply:
I'm just wondering if we should do this during a release or in a separate step?

Reply:
Its a manual action to check if the tarball can be build, as it doesn't take much time it could also be checked in the main project (so you always know if you can release)

The tarball here is not the tarball which will end up on the domjudge.org server, so its more for convenience.

We can also run this in a `matrix`, and use the tarball as an artifact and do some more extensive testing so most things don't have to be done locally.

So what kind of separate step do you mean?

Reply:
Nope then it's fine!